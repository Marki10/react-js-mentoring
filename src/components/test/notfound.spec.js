import React, { Component } from 'react';
import NotFound from '../notfound';
const ReactTestRenderer = require('react-test-renderer');

describe('Not found', () => {
    it('Render not found page', () => {
        const component = ReactTestRenderer.create(
            <NotFound />
        );
        const testTemp = component.toJSON();
        expect(testTemp).toMatchSnapshot();
    })
});