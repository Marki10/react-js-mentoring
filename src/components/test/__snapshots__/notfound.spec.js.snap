exports[`Not found Render not found page 1`] = `
<section
  className="container not-found">
  Not Found
</section>
`;
